modules/influxdb/main.tf

```diff
@@ -0,0 +1,17 @@
+# https://github.com/bitnami/charts/tree/3f2b19ad2743d28828b5c217b2b682e0db57be75/bitnami/influxdb
+resource "helm_release" "influx" {
+  name       = "influx"
+  repository = "https://charts.bitnami.com/bitnami"
+  chart      = "influxdb"
+  version    = "1.1.8"
```

Comment:
Change to
```
version = "~>1.1.8"
```

Reply:
This would change chart version into 1.1.9 which cannot be viewed

Reply:
Gotcha! For future reference this is because of decision to remove [values-production.yaml](https://github.com/bitnami/charts/commit/67be85057198bcdf4a17fdbd067018a7cbb7b7c4) on Bitnamis behalf.